docs/book/src/user/quick-start.md

```diff
@@ -125,6 +125,22 @@ Test to ensure the version you installed is up-to-date:
 ```
 clusterctl version
 ```
+{{#/tab }}
+{{#tab homebrew}}
+
+##### Install clusterctl with homebrew on macOS and linux
+
+Install the latest release using homebrew:
+
+```bash
+brew install clusterctl
+```
```

Comment:
One quick question on this:

- How are we going to handle upgrading within a release series? Like upgrade between v0.3.14 to v0.3.15 for example.
- How can we support multiple release series? Like 0.3 for now, and 0.4 later.

Reply:
Going to open a PR on #2389

Regarding multiple release, homebrew supports installation like `brew install kubectl@0.3` out of the box.

Reply:
Sorry wrong issue #4356 :D